test/integration/test-amp-bind.js

```diff
@@ -20,587 +20,606 @@ import {FormEvents} from '../../extensions/amp-form/0.1/form-events';
 import {Services} from '../../src/services';
 import {createFixtureIframe} from '../../testing/iframe';
 
-describe.configure().ifNewChrome().run('amp-bind', function() {
-  // Give more than default 2000ms timeout for local testing.
-  const TIMEOUT = Math.max(window.ampTestRuntimeConfig.mochaTimeout, 4000);
-  this.timeout(TIMEOUT);
-
-  let fixture;
-  let sandbox;
-  let numSetStates;
-  let numTemplated;
-
-  beforeEach(() => {
-    sandbox = sinon.sandbox.create();
-    numSetStates = 0;
-    numTemplated = 0;
-  });
-
-  afterEach(() => {
-    fixture = null;
-    sandbox.restore();
-  });
-
-  /**
+describe.configure()
+    // This can be turned back on when amp-bind works at all with
```

Comment:
what causes amp-bind to not work? the generated parser code?

Reply:
Fix in #16934